### ISSUE TITLE
LibWeb: Something is wrong with height calculation inside flex item

### ISSUE THREAD

**Issue by kalenikaliaksandr** [SerenityOS/serenity#21054]
Reduction from https://linktr.ee/katalinkult:
```html
<!DOCTYPE html><style type="text/css">
  * {
    border: 1px solid black;
  }
  .flex-item {
    flex-grow: 1;
  }
  .box {
    height: 100%;
  }
  .flex-container {
    display: flex;
    min-height: 100px;
    flex-direction: column;
  }
</style><div class="flex-container"><div class="flex-item"><div class="box">I make music you can dream to.</div><div>My SoundCloud</div></div></div>
```

## Funding

- We're currently trying [Polar.sh](https://polar.sh/SerenityOS) where you can help fund the work on this issue.
- Contributor(s) receive the funding once the issue is completed & confirmed.

<a href="https://polar.sh/SerenityOS/serenity/issues/21054">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/SerenityOS/serenity/issues/21054/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/SerenityOS/serenity/issues/21054/pledge.svg">
</picture>
</a>

**Comment by tcl3** [comment]
The reduction now appears to render correctly. The commit that fixed this appears to be f963bb4f3632358a79fc3744ee7c8eb141e1662b. 

Does that mean this issue can be closed?

**Comment by awesomekling** [comment]
Oh hey, nice that we got it fixed. Yes, then we can close it! :)
